packages/pharos/src/components/image-card/pharos-image-card.test.ts

```diff
@@ -609,4 +609,33 @@ describe('pharos-image-card', () => {
 
     expect(component.innerHTML).contains('Card overlay');
   });
+
+  it('the checkbox dispays when tab is pressed on the image link', async () => {
```

Comment:
I've realized this test does not do what I thought it does. Working on replacing it.

Reply:
The reason my tests are not working is because any keyboard events from the test have `isTrusted` set as false. This is making the tests fail.